src/native/eventpipe/ep-file.c

```diff
@@ -304,6 +313,11 @@ ep_file_alloc (
 	StreamWriter *stream_writer,
 	EventPipeSerializationFormat format)
 {
+	dn_umap_custom_alloc_params_t params = DN_UMAP_DEFAULT_ALLOC_PARAMS;
```

Comment:
I think we should make it part of the style that you can write
```c
dn_umap_custom_alloc_params_t params = {0,};
```

And explicitly ensure that zero-initialized members of the params struct imply default values.

I don't like the `DEFAULT_ALLOC_PARAMS` macros.

Reply:
ok, lets say 0 is default for all members.